### DISCUSSION TITLE
Certain languages not showing up in my Most Used Languages Card (Repos I don't own?)

### DISCUSSION THREAD

**Discussion by HRosser15** [anuraghazra/github-readme-stats#3753] [Q&A] [answered]
I recently wrote a couple thousand lines of code in JSX, a few hundred in JavaScript, about a thousand in Java, and a couple hundred in CSS for an organization's repositories. However, these languages are not appearing in my Most Used Languages Card.

Additional details:
- The repositories in the organization are public (they were private until about 30 minutes ago-which is my best guess as to why they aren't appearing).
- I made thousands of lines of contributions directly to the repositories (not forks) and merged them into the main branch.
- I have verified that the repositories are public and my contributions are visible on GitHub.
- I'm using the default Top Languages Card configuration from the GitHub Readme Stats project.

I also wrote a few hundred codes of JSX in another repository a few months ago (this was to a public repo owned by another student I was working with, but I made contributions that were merged into main).

Is there any additional configuration required to include these contributions in the Most Used Languages Card?

This is what I see in my README:
![image](https://github.com/anuraghazra/github-readme-stats/assets/117858984/da7e637a-7def-499e-8d03-2468f4a5e511)

This is what I see when visiting https://ionicabizau.github.io/github-profile-languages/ :
![image](https://github.com/anuraghazra/github-readme-stats/assets/117858984/f28e6a35-114c-4f6b-a287-a704327a3ba2)

I have also looked at the answers to a few related discussions, and from this one:  https://github.com/anuraghazra/github-readme-stats/discussions/1878
the repsonse to the query does NOT show the repos I am not the OWNER of. However, when removing that argument, it does show the repositories I contributed to. Not sure if the languages only show up for repos I am the owner of. If so, is there a way around this?

Any insights or assistance would be greatly appreciated. Thank you!

**Comment by qwerty541** [comment, marked as answer]
Hey @HRosser15.

From our documentation:

The card shows language usage only inside your own non-forked repositories, not depending on who the author of the commits is. It does not include your contributions into another users/organizations repositories. Currently there are no way to get this data from GitHub API. If you want this behavior to be improved you can support [this feature request](https://github.com/orgs/community/discussions/18230) created by [@rickstaa](https://github.com/rickstaa) inside GitHub Community.

By default, the language card shows language results only from public repositories. To include languages used in private repositories, you should [deploy your own instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) using your own GitHub API token.

**Reply by HRosser15** [reply]
Thank you! Makes sense that is how it works. 

I'll look into those links.

**Reply by qwerty541** [reply]
Glad to help! Please mark this discussion as answered if you don't need assistance anymore.
